content/en/security/application_security/threats/security_signals.md

```diff
@@ -0,0 +1,89 @@
+
+## Triage a signal
+
+You can triage a signal by assigning it to a user for further investigation. The assigned user can then track their review by updating the signal's status.
+
+1. On the [Signals Explorer][2], select a security signal.
+2. On the signal side panel, click the user profile icon and select a user.
+3. To update the status of the security signal, click the triage status dropdown menu and select a status. The default status is **Open**.
+    - **Open**: The signal has not yet been resolved.
+    - **Under Review**: The signal is actively being investigated. From the **Under Review** state, you can move the signal to **Archived** or **Open** as needed.
+    - **Archived**: The detection that caused the signal has been resolved. From the **Archived** state, you can move the signal back to **Open** if it's within 30 days of when the signal was originally detected.
+
+**Note**: To modify security signals, you must have the `security_monitoring_signals_write` permission. See [Role Based Access Control][9] for more information about Datadog's default roles and granular role-based access control permissions available for Application Security Management.
+
+## Create a case
+
+Use [Case Management][6] to track, triage, and investigate security signals.
+
+1. On the [Signals Explorer][2], select a security signal.
+2. On the signal side panel select the **Create a case** dropdown. Select **Create a new case**, or **Add to an existing case** to add the signal to an existing case. 
+3. Enter a title and optional description.
+4. Click **Create Case**.
+
+## Declare an incident 
+
+Use [Incident Management][4] to create an incident for a security signal.
+
+1. On the [Signals Explorer][2], select a security signal.
+2. On the signal side panel, click the **Declare Indident** dropdown menu and select **Create an incident**, or **Add to an existing incident**.
+3. On the incident creation modal, configure the incident by specifying details such as the severity level and incident commander.
+4. Click **Declare Incident**.
+
+## Run a workflow
+
+Use [Workflow Automation][5] to manually trigger a workflow for a security signal. 
+
+1. On the [Signals Explorer][4], select a security signal.
+2. Scroll down to the **What is Workflow Automation** section. 
+3. Click **Run Workflow**.
+4. On the workflow modal, select the workflow you want to run. Depending on the workflow, you may be required to enter additional input parameters.
+5. Click **Run**.
+
+## Review and remediate
+
+1. On the [Signals Explorer][2], select a security signal.
+2. On the signal side panel, click each of the review tabs such as **Attack Flow**, **Activity Summary**, and **Rule Details**.
+3. Review the **Suggested Next Steps**, and take action:
+    -  Click to **Block attacking users and IPs**
+    -  Click to **Block with Edge WAF**.
```

Comment:
Below line 75 add
- Click to **Automated Attacker Blocking**

^ this is new

Reply:
thanks, I will update the screen shot as well!